public/scripts/app.js

```diff
@@ -0,0 +1,29 @@
+angular
+  .module('booksApp', ['ngRoute']) // ngRoute allows us to configure client-side routes in our app.
+  .config(config); // for the app, use this dependency and configuration
+
+/* ROUTES */
+
+  config.$inject = ['$routeProvider', '$locationProvider'];
```

Comment:
This shouldn't be indented, and it throws off you indentation for the rest of the file.